src/python/CRABInterface/Regexps.py

```diff
@@ -98,7 +98,7 @@
 RX_ASODB = RX_USERNAME
 
 ## need to be careful with this
-RX_TEXT_FAIL = re.compile(r"^.*{0,10000}$")
+RX_TEXT_FAIL = re.compile(r"^.{0,10000}$")
```

Comment:
wouldn't it be better to use a (hopefully already-existing), sanely-named regexp to validate the warning message as a string, instead of keeping everybody wonder that the hell TEXT_FAIL may mean here ?
i.e. change the line in RESTTtask https://github.com/dmwm/CRABServer/blob/466bcba95472b1b256c087016ae37c572d9a5f38/src/python/CRABInterface/RESTTask.py#L37

Reply:
We have 

```python
RX_ANYTHING = re.compile(r"^.*$")
```
which does not limit length.

I understand why we could want to limit the length of a warning/failing message. Maybe we can call this new regex

```python
RX_ANYTHING_SHORT = re.compile(r"^.{0,10000}$")
```

?

Reply:
I'd for for RX_ANYTHING, it is already (ab)used in a lot of places even for short strings, if we need a length check we need to do it independently. I am not sure that we really gain by all those REGEXP;s other then where they help as format documentation.

Reply:
The last commit contains this suggestion. I will mark this conversation as resolved. I would not merge this yet since i would like to test it, tomorrow or thursday at the latest.